input/fsh/au-erequesting-imaging-request.fsh

```diff
@@ -0,0 +1,23 @@
+Profile: AUeRequestingImagingServiceRequest
+Parent: AUeRequestingServiceRequest
+Id: au-erequesting-imaging-request
+Title: "AU eRequesting ImagingRequest"
+Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about diagnostic imaging requests, plans, or orders for a service for a patient. It is based on the [AU eRequesting ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
+
+
+* ^status = #draft
+
+* ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
+
+* . ^short = "A diagnostic imaging request"
+* . ^definition = "A request for a medical or diagnostic imaging service."
```

Comment:
remove 'medical or' from the definition.

Reply:
Done